v2/robotmk/src/attempt.rs

```diff
@@ -215,7 +213,7 @@ mod tests {
                 name: "suite_1",
                 timestamp: "2023-08-29T12.23.44.419347+00.00".into(),
             },
-            working_directory: &Utf8PathBuf::from("/tmp/outputdir/"),
+            working_directory: &Utf8PathBuf::from("/tmp/outputdir/suite_1"),
```

Comment:
What does this change mean?

Reply:
`RetrySpec` does not add the suite name anymore when computing `output_directory`, since this already part of `Suite.working_directory` and handed over to `RetrySpec.working_directory`. That's why I added it here.